readmes/README.Linux.md

```diff
@@ -6,23 +6,33 @@
   ./README.CentOS.bash
   ```
 
-- If you want to link cmake3 to cmake, run:
+- Update gcc to (devtoolset-10) to support c++14
 
   ```bash
-  sudo ln -sf /usr/bin/cmake3 /usr/local/bin/cmake
+  yum install centos-release-scl
+  yum -y install devtoolset-10-gcc devtoolset-10-gcc-c++ devtoolset-10-binutils
+  scl enable devtoolset-10 bash
+  source /opt/rh/devtoolset-10/enable
   ```
 
-- Make sure that you add `/usr/local/lib` and `/usr/local/lib64` to
-`/etc/ld.so.conf`, then run command `ldconfig`.
+- Install additional packages required for some configurations
 
-- If you want to install and use gcc-7 by default, run:
+  ```bash
+  yum -y install R apr apr-devel apr-util automake autoconf bash bison bison-devel bzip2 bzip2-devel centos-release-scl curl flex flex-devel gcc gcc-c++ git gdb ibxml2 iproute krb5 krb5-devel less libcurl libcurl-devel libevent libevent-devel libxml2 libxml2-devel libyaml libzstd-devel libzstd.x86_64 make openldap openssh-client openssl openssl-devel openssl-libs perl python3-devel readline readline-devel rsync sed sudo tar vim wget which xerces-c-devel zip zip-devel zlib && \
+  yum -y install epel-release.noarch && \
+  yum -y install libzstd.x86_64 && \
+  yum -y install devtoolset-9-gcc*
+  ```
+  
+- If you want to link cmake3 to cmake, run:
```

Comment:
I communicated with my development classmates and found that this is a solution to prevent the system from downloading cmake3 but not using it, which needs to be retained.